src/model.js

```diff
@@ -2538,7 +2538,7 @@ class Model {
       const createOptions = { ...options };
 
       // To avoid breaking a postgres transaction, run the create with `ignoreDuplicates`.
-      if (this.sequelize.options.dialect === 'postgres' && options.transaction) {
+      if ((this.sequelize.options.dialect === 'postgres' || this.sequelize.options.dialect === 'yugabyte') && options.transaction) {
```

Comment:
Just for knowledge sake, what does ignoreDuplicates do?

Reply:
It is an option for inserts to ignore if there is any duplicates which is same as `ON CONFLICT DO NOTHING`.